libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.ts

```diff
@@ -295,16 +308,7 @@ export class DhWholesaleRequestCalculationComponent {
       : [];
   }
 
-  private getExcludedProcessTypes(selectedEicFunction: SelectedEicFunctionType) {
-    if (
-      selectedEicFunction === EicFunction.BalanceResponsibleParty ||
-      selectedEicFunction === EicFunction.EnergySupplier
-    )
-      return wholesaleCalculationTypes;
-
-    if (selectedEicFunction === EicFunction.SystemOperator)
-      return [CalculationType.Aggregation, CalculationType.BalanceFixing];
-
-    return [];
+  hasPermission(permission: Permission) {
+    return toSignal(this._permissionService.hasPermission(permission));
```

Comment:
suggestion: maybe add options with `{ initialValue: false }` so that the signal becomes of type `Signal<boolean>` instead of `Signal<boolean | undefined>`.

Reply:
I just went ahead and fixed it:
d6ec0066f1a86e630c7da923a275e8aa6f8c62f2